.github/workflows/tests-e2e.yml

```diff
@@ -63,24 +63,46 @@ concurrency:
 
 jobs:
   e2e:
-    name: '${{ matrix.browser }} - WP ${{ matrix.wp }}'
+    name: '${{ matrix.browser }} - WP ${{ matrix.wp }} (${{ matrix.shard }})'
     runs-on: ubuntu-latest
     timeout-minutes: 30
     continue-on-error: ${{ matrix.experimental == true }}
     strategy:
       matrix:
         # TODO: add back Firefox once support is more mature.
+        browser: ['chrome']
+        wp: ['latest', '5.5']
+        snapshots: [false]
+        # We want to split up the tests into 2 parts running in parallel.
+        shard: ['1/2', '2/2']
         include:
           - browser: 'chrome'
             wp: 'latest'
             snapshots: true
+            shard: '1/2'
+          - browser: 'chrome'
+            wp: 'latest'
+            snapshots: true
+            shard: '2/2'
           - browser: 'chrome'
             wp: '5.9-RC2'
             experimental: true
             snapshots: false
+            shard: '1/2'
           - browser: 'chrome'
-            wp: '5.5'
+            wp: '5.9-RC2'
+            experimental: true
+            snapshots: false
+            shard: '2/2'
+        exclude:
+          - browser: 'chrome'
+            wp: 'latest'
+            snapshots: false
+            shard: '1/2'
+          - browser: 'chrome'
+            wp: 'latest'
             snapshots: false
+            shard: '2/2'
```

Comment:
Just for my own education: why do we have to build each of these out separately?

Reply:
Hmm I can probably simplify this.

Basically we want to run these tests:

* WP 5.5, no snapshots
* WP latest, with snapshots
* WP 5.9 RC, no snapshots, experimental

So this involves using `include` here to add the additional tests.

Thanks to your comment I realized that I could simplify this matrix a bit though.